wazimap_np/templates/profile/sections/demographics.html

```diff
@@ -110,6 +110,33 @@ <h2 class="header-for-columns"><a class="permalink"
                 </section>
                 <section class="clearfix stat-row">
                     <h2 class="header-for-columns"><a class="permalink"
+                                                      href="#citizenship"
+                                                      id="citizenship">Citizenship <i 
+                                                          class="fa fa-link"></i></a></h2>
+                    <div class="column-half"
+                       id="chart-pie-demographics-citizenship_distribution"
+                       data-stat-type="percentage"></div>
+
+                    <div class="column-half">
+                       <div id="chart-grouped_column-demographics-citizenship_by_sex"
+                           data-stat-type="scaled-percentage"
+                           data-chart-title="Population by citizenship and sex"
+                           data-qualifier="Universe: {{ demographics.citizenship_distribution.metadata.universe }}"></div>
+                    </div>
+                    <small>Sources:
+                        <br>
+                        <a target="_blank"
+                           href="http://data.opennepal.net/content/population-citizenship-and-sex-2011-12">OpenNepal
+                            Population by citizenship and sex
+                            (2011-12)</a>
+                        <br>
+                        <a target="_blank"
+                           href="https://github.com/opennepal/odp-census/blob/master/%20Population%20by%20citizenship%20and%20sex/data.csv?download=1">National
```

Comment:
The original PDF is here: https://github.com/opennepal/datasources/blob/census_population/Census/Volume05Part01.pdf
Table 9 holds the data from which the CSV was taken.

I suggest we update this because the PDF is the official document.

(I wasn't sure which part of the census it would be found in since there are several PDFs which each have part of the census.